doc/guides/dts/howtos.rst

```diff
@@ -204,21 +204,35 @@ variable :makevar:`DTC_OVERLAY_FILE` contains a space- or semicolon-separated
 list of overlays. If :makevar:`DTC_OVERLAY_FILE` specifies multiple files, they
 are included in that order by the C preprocessor.
 
-Here are some ways to set it:
-
-1. on the cmake build command line
-   (``-DDTC_OVERLAY_FILE="file1.overlay;file2.overlay"``)
-#. with the CMake ``set()`` command in the application ``CMakeLists.txt``,
-   before including zephyr's :file:`boilerplate.cmake` file
-#. create a ``boards/<BOARD>_<revision>.overlay`` file in the application
-   folder for the current board revision. This requires that the board supports
+Devicetree overlays can be specified by the user directly, in a
+:file:`CMakeLists.txt` file, or applied automatically.
+
+If an overlay is specified by the user or directly in a :file:`CMakeLists.txt`
+file then no overlay files will be applied automatically.
```

Comment:
The two sentences above are redundant with the description below. It should be removed IMHO.

It's better to say something once unambiguously than to say it in different ways with varying degrees of precision.

Reply:
> It's better to say something once unambiguously than to say it in different ways with varying degrees of precision.

What I would like to avoid is confused readers, like here:
https://github.com/zephyrproject-rtos/zephyr/issues/38251

Especially this comment:
> Devicetree HOWTOs describe some ways to set DTC_OVERLAY_FILE. No where it states they are mutually exclusive 

Of course the description on how the files are processed should make it clear, just think this important enough to highlight in an extra sentence with specific purpose.

@marinjurjevic do you have any thoughts on this. I'm open to drop the extra sentence.

Reply:
a
otherwise b
otherwise c

is now clearly communicating it is mutual exclusive IMHO.

Reply:
> is now clearly communicating it is mutual exclusive IMHO.

I tend to agree with you, but as i'm not the target audience of this text I would prefer some feedback from a golden `new-comer` user.
And as @marinjurjevic was raising #38251 I see him as the golden users :wink:

Reply:
I think having `Otherwise` is the key so there is no need for extra sentence. `Otherwise` should make it clear that listed options are mutually exclusive

Reply:
Thanks @SebastianBoe  and @marinjurjevic.
Will update accordingly.

Reply:
Fixed.